### ISSUE TITLE
switch from inputBinding to arguments: based syntax

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/user_guide#34]

**Comment by tobyhodges** [comment]
To be clear: this is for all `inputBinding` fields? Or only specific examples/use cases?

**Comment by mr-c** [comment]
Yes, for everything except arrays it is clearer or simpler to use the `arguments` style

It will be a bit awkward when it comes time to explain arrays, but I think it is worth it.

**Comment by tobyhodges** [comment]
ok, this calls for a bigger PR then. Hope to have time in coming days.

**Comment by swzCuroverse** [comment]
@kinow - Did you do this already for the examples or do we need to do this still.  If so, this might be a starting ticket since it is pretty straightforward.

**Comment by kinow** [comment]
I think there was one place I did that, but I think it was in the new examples. But I didn't go through the rest of the user guide to see what else could be updated. That's still pending :+1:
